From 406b06e90c19beb05cbcfa9ce62ffbf16555b057 Mon Sep 17 00:00:00 2001
From: subbareddy <gajulapalli001@gmail.com>
Date: Wed, 28 Jun 2017 10:14:41 +0530
Subject: [PATCH 2/3] some file modified

---
 login.php | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/login.php b/login.php
index ffdbe8b..c243dce 100644
--- a/login.php
+++ b/login.php
@@ -2,5 +2,7 @@ New php file ()
 Hello 
 
 {
-Void Main 
+public Void Main(string args);
+system.out.println("ha------------");
+ 
 }
-- 
2.7.4

